app/scraper/sources/face_book.py

```diff
@@ -234,3 +234,4 @@ def scrape(self, current_people):
             people = self.compare_years(backup, people, emails)
 
         self.new_records = people
+        self.image_uploader.deleted_unused_imgs(people)
```

Comment:
Hmm... I'm not quite sure about this. My concern is that if the scraper doesn't complete successfully due to a failure in another part (like, if the directory scraper crashes or something), then we would have removed images that are still being used. Even if it doesn't crash, there would be a decent amount of time between this running and actually inserting these new records into the database, during which time a bunch of people would have missing photos. It may be better to execute this at the very end.